### DISCUSSION TITLE
对 NFC 的一些说明和建议

### DISCUSSION THREAD

**Discussion by dangfan** [canokeys/canokey-product#4] [Ideas]
受芯片功耗的限制，CanoKey Pigeon 的 NFC 性能和 YubiKey 确实无法相比。出厂前， @cnZary 使用典型的读卡器 ACR 122u 对所有的 Key 都做了 WebAuthn 的 enrollment 计算测试，但这样的测试对众多的手机仍然是不够的，并且 Key 之间还有一些个体差异（比如沉铜的厚度、阻焊的厚度等等），所以表现出来的灵敏度确有差异。

以下是一些可能有助于提升 NFC 可用性的建议：

1. 对于所有的手机（包括 Android 和 iPhone），请不要来回找位置，这是因为一旦程序没有启动，但又有电，会因为低电压保护而无法启动。寻找最佳读取位置的时候，请快速贴在手机上，如果不行，请完全拿开，找下一个位置。
2. 对于 iPhone，请尝试以下两个位置：听筒正上方（即完全横着盖住听筒）和摄像头旁边紧贴手机（同样需要横着）。由于 iOS 15 iPhone 修改了寻卡的策略，所以如果看不到提示，可以稍等片刻重新尝试。
3. 对于 Android，由于不同手机天线位置差异很大，因此请遵循第一条的方法寻找最佳位置。建议开启系统声音，NFC 寻卡有标准的声音，利用声音可以快速判断位置是否正确。
4. 任何时候，请勿测试 NFC 极限性能（即读卡距离），原因请见第一条，这样的测试通常都会失败。

**Comment by sorz** [comment]
另见 #6 (NFC修复指南)

**Comment by 70599** [comment]
iPhone扫完NFC后，会弹出“网站NFC标签”，提示要打开`canokeys.org`，这个可以关掉吗？

**Reply by cnZary** [reply]
在webconsole中关闭ndef即可

**Reply by 70599** [reply]
可以了，谢谢
